packages/core/src/components/dropdown-item/dropdown-item.scss

```diff
@@ -8,6 +8,8 @@
  */
 
 @import 'common-variables';
+@import 'fonts';
```

Comment:
This imports all fonts avaiable for components not optimal will increase scss.

Better use mixin or `ix-typography`